### PR TITLE
mutanus 0.1.x (new formula)

### PR DESCRIPTION
**Pull request by SoriUR**
Mutanus is a command line tool written in Swift dedicated to perform mutation testing of Swift projects. 

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/mutanus.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  homepage "https://github.com/SoriUR/mutanus/"
+  url "https://github.com/SoriUR/mutanus/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "73204fe4b1788c2427d2d12f8a5d2579469cd8910ecfa777974d431118d83155"
+  license "MIT"
+
+  depends_on xcode: [">= 12.5", :build]
+  depends_on macos: [
+    :catalina,
+    :big_sur,
+  ]
```
Comment (carlocab) [inline Formula/mutanus.rb:12]:
```suggestion
  depends_on xcode: ["12.5", :build]
  depends_on macos: :catalina
```

File: Formula/mutanus.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    bin.install ".build/release/mutanus"
+  end
+
+  test do
+    system "#{bin}/mutanus", "--help"
```
Comment (carlocab) [inline Formula/mutanus.rb:20]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Comment (SoriUR) [reply Formula/mutanus.rb:17]: I didn't manage to write another test. This tool requires a swift project to run properly and has no other commands.
I can potentially create a command just for this test. Or I can try to create test project, download it, and execute the test

Comment (SMillerDev) [reply Formula/mutanus.rb:17]: The latter seems good.

Comment (SoriUR) [reply Formula/mutanus.rb:17]: I've tried this
```
class Mutanus < Formula
  desc "Performs mutation testing of you Swift project"
  homepage "https://github.com/SoriUR/mutanus/"
  url "https://github.com/SoriUR/mutanus/archive/refs/tags/v0.1.1.tar.gz"
  sha256 "a534071a0f6f234cd3dc412afab6e825d54181fd8de0a8cc2a3478f1ba2189a8"
  license "MIT"

  depends_on xcode: [">= 12.5", :build]
  depends_on macos: :catalina

  resource("testdata") do
    url "https://github.com/SoriUR/MutanusTest/archive/refs/tags/v1.0.1.zip"
    sha256 "4680528a2697ca2a9f956b2c2756d29bb350ceabf1ec208cdef4ff5fce271cb9"
  end

  def install
    system "swift", "build", "--disable-sandbox", "-c", "release"
    bin.install ".build/release/mutanus"
  end

  test do
      # download test project    
    resource("testdata").stage do
      # download simulator
      system "xcrun simctl create \"iPhone\" \"iPhone 11 Pro Max\" iOS14.5"  
      # building test project
      system "xcodebuild test -project MutanusTest.xcodeproj -scheme MutanusTest -destination \"platform=iOS Simulator,name=iPhone 12 Pro Max\""

    end
  end
end
```

but got error
```
Testing failed:
	Couldn't create workspace arena folder '/Users/DD/MutanusTest-gyxuazfdcgumyheydlytxkovhuyu': You don’t have permission to save the file
```
	
Do you have any ideas?

File: Formula/mutanus.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+class Mutanus < Formula
+  desc "Performs mutation testing of you Swift project"
+  homepage "https://github.com/SoriUR/mutanus/"
+  url "https://github.com/SoriUR/mutanus/archive/refs/tags/v0.1.2.tar.gz"
+  sha256 "ec5f663a76166c18ee3a76e3fd69bfd1b29526dc567c3994e9a119624b992e34"
+  license "MIT"
+
+  depends_on xcode: [">= 12.5", :build]
```
Comment (carlocab) [inline Formula/mutanus.rb:8]:
```suggestion
  depends_on xcode: ["12.5", :build]
```

**Review by carlocab — CHANGES_REQUESTED** [review]
The `>=` is redundant and isn't even parsed by `brew`. See https://github.com/Homebrew/brew/blob/998a4028cf2678098c6e76b002aa3657c14bbf0a/Library/Homebrew/requirements/xcode_requirement.rb#L18-L21.

Please remove it.

**Pull request closed without merging**